app/assets/javascripts/avalon.js

```diff
@@ -25,7 +25,8 @@ $(document).ready(function() {
       var longest = $(this).find('.facet-count span').sort(longer).first();
       var clone = longest.clone().css('visibility','hidden');
       $('body').append(clone);
-      $(this).find('.facet-count').first().width(clone.width());
+      // U of A: add 2 pixel fudge factor to avoid bad number wrapping
+      $(this).find('.facet-count').first().width(clone.width()+2);
```

Comment:
This seems insane? They are using bootstrap panels for the facet themselves, why not do this smart? Like using list-groups and badges or something (which are floated right and will automatically expand accordingly)? 
http://getbootstrap.com/components/#panels-list-group
http://getbootstrap.com/components/#list-group-badges

Instead they wrote all custom css stuff using `display: table-cell` and hacking this in javascript to get this to look nice....sigh :-1: .

Might be more work to do it correctly. So this "hack" is fine.

Only complaint is your magic number of `2`. Maybe assign this to a variable so this is easily readable and refactorable for the next developer guessing what this special number is for? More info about this here: http://eslint.org/docs/rules/no-magic-numbers